clustering/singleton/api/src/main/java/org/wildfly/clustering/singleton/SingletonServiceBuilderFactory.java

```diff
@@ -29,9 +29,11 @@
  * @author Paul Ferraro
  */
 public interface SingletonServiceBuilderFactory extends SingletonPolicy {
-    /** Use {@link org.wildfly.clustering.singleton.spi.SingletonServiceName} to generate an appropriate {@link ServiceName}.
+    /**
+     * Use {@link org.wildfly.clustering.singleton.SingletonServiceName} to generate an appropriate {@link ServiceName}.
+     */
     @Deprecated
-    ServiceName SERVICE_NAME = ServiceName.JBOSS.append("clustering", "singleton", "builder");
+    ServiceName SERVICE_NAME = ServiceName.JBOSS.append("clustering", "singleton");
```

Comment:
Can also remove it for good, now that it's been efectively removed in 10.0.0.Final :-)